src/CSE.cpp

```diff
@@ -58,11 +58,6 @@ bool should_extract(Expr e) {
         return !is_const(a->stride);
     }
 
-    if (const Call *a = e.as<Call>()) {
```

Comment:
I think we still shouldn't lift loads out of address_of. address_of is broken if it's operand isn't a Load. predicated load/store isn't the only possible use of this...